### PR TITLE
Update link to shunit2

### PR DESCRIPTION
**Pull request by nichtich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ brew install jqnpm
 **On other systems**
 
 - Clone or download, then symlink `src/jqnpm`. There is no build step.
-- Requirements: [jq](https://stedolan.github.io/jq/) 1.5+, [bash](https://www.gnu.org/software/bash/) 4+, [git](http://git-scm.com/), [shUnit2](https://code.google.com/p/shunit2/).
+- Requirements: [jq](https://stedolan.github.io/jq/) 1.5+, [bash](https://www.gnu.org/software/bash/) 4+, [git](http://git-scm.com/), [shUnit2](http://github.com/kward/shunit2).
```
Comment (joelpurra) [inline README.md]: Link should not be downgraded to `http`; please use `https`.

**Comment by joelpurra** [comment]
Ah, they moved due to Google Code shutting down -- thanks for the patch! Have one comment; please fix before merge.

Btw, can you confirm that recent `shunit2` versions still work well?

**Comment by nichtich** [comment]
Ok, I've updated the link and going through the instructions.

**Comment by joelpurra** [comment]
@nichtich: great, thanks for the contribution! =)

**Pull request merged**